Source/WebCore/platform/xr/PlatformXR.h

```diff
@@ -368,7 +368,7 @@ class Device : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device> {
 
     void ref() const { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device>::ref(); }
     void deref() const { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device>::deref(); }
-    ThreadSafeWeakPtrControlBlock& controlBlock() const { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device>::controlBlock(); }
+    ThreadSafeWeakPtrControlBlock& controlBlock() const { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device>::controlBlock(); }    size_t weakRefCount() const { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Device>::weakRefCount(); }
```

Comment:
Ditto.  Also, I think the &lt;Device&gt; isn't needed in all these cases.  Could be a nice little cleanup.

Reply:
It seems like it shouldn't need to be defined at all since they're not virtual anyway. I removed them.